libraries/AC_PrecLand/AC_PrecLand_IRLock.cpp

```diff
@@ -54,3 +54,9 @@ uint32_t AC_PrecLand_IRLock::los_meas_time_ms() {
 bool AC_PrecLand_IRLock::have_los_meas() {
     return _have_los_meas;
 }
+
+// return target ID
+uint8_t AC_PrecLand_IRLock::target_num()
+{
+    return 0;
+}
```

Comment:
miss newline at the file end (it help for diff tool)

Reply:
Fixed in https://github.com/ArduPilot/ardupilot/pull/9134/commits/6316dbeb7a8e16fe25994300712cb2130fbdeb3d